### ISSUE TITLE
When running in Node.js method File.read returns an "ENOENT: no such file or directory" error

### ISSUE THREAD

**Issue by ggrossetie** [asciidoctor/asciidoctor-template.js#9]
As a consequence the following code is not working because we are catching `IOError`: 

```ruby
def try_read name
  ::File.read(name)
rescue IOError
  # Ignore error as Asciidoctor expect a nil value if the template is not found
  nil
end
```
https://github.com/asciidoctor/asciidoctor-template.js/blob/8e68e70ccf1d134956875043baefd503e0d18f0f/lib/asciidoctor/core_ext/template.rb#L45-L50

When running in Node.js `ENOENT` is not "cast" as an `IOError` by Opal.

https://nodejs.org/api/errors.html

**Comment by ggrossetie** [comment]
Asking Opal's guru about this one to know how to fix it...

**Comment by ggrossetie** [comment]
EONENT is now cast as an `IOError` in Opal: https://github.com/opal/opal/pull/1615
This fix will be available in Opal 0.11

**Comment by ggrossetie** [comment]
Fixed in https://github.com/asciidoctor/asciidoctor-template.js/tree/vv1.5.5-integration2
